examples/replay/replay_mover.py

```diff
@@ -156,17 +166,21 @@ def run(self, job_id):
     def move_single_dataset(self, job_id, cycle):
         """Store a single cycle to zarr"""
 
-        replay = FV3Dataset(path_in=self.cached_path, config_filename=self.config_filename)
+        replay = self.Dataset(path_in=self.cached_path, config_filename=self.config_filename)
         xds = replay.open_dataset(cycle, **self.ods_kwargs(job_id))
 
```

Comment:
do we need to call the regridder here if its mom6 of cice? I might be confusing myself but as of right now it this just opening the mom6 nc file and writing it to zarr as is without regridding?

Reply:
Yeah we will want to call the regridder, e.g. `if "regrid" in replay.config.keys()` or something like that. I didn't add any of the regridding calls, I was just wanting to generalize the moving. Could you add a call to the regridder in #18? I think it's more appropriate for the scope of that PR

Reply:
@mariahpope if you agree with adding the regridding lines to #18 and if you agree that this PR is otherwise good to go then I will go ahead and merge so that you have a clean slate in #18 to work with. Let me know what you think